test/parallel/test-module-loading-error.js

```diff
@@ -22,23 +22,40 @@
 'use strict';
 const common = require('../common');
 const assert = require('assert');
+const { execSync } = require('child_process');
 
 const error_desc = {
   win32: ['%1 is not a valid Win32 application'],
   linux: ['file too short', 'Exec format error'],
   sunos: ['unknown file type', 'not an ELF file'],
   darwin: ['file too short']
 };
-const dlerror_msg = error_desc[process.platform];
+// if we don't know a priori what the error would be, we accept anything
+const dlerror_msg = error_desc[process.platform] || [''];
+
+// On Windows error message are MUI dependent
+// Ref: https://github.com/nodejs/node/issues/13376
+let localeOk = true;
+if (common.isWindows) {
+  const powerShellFindMUI =
+    'powershell -NoProfile -ExecutionPolicy Unrestricted -c' +
+    ' "(Get-UICulture).TwoLetterISOLanguageName"';
+  try {
+    localeOk = execSync(powerShellFindMUI).toString('utf8') === 'en';
+  } catch (_) {
+    // it's a best effort try to find the MUI
+  }
+}
+
 
 assert.throws(
-  () => { require('../fixtures/module-loading-error.node'); },
+  () => {
+    require('../fixtures/module-loading-error.node');
+  },
   (e) => {
-    if (dlerror_msg && !dlerror_msg.some((msg) => e.message.includes(msg)))
-      return false;
-    if (e.name !== 'Error')
+    if (localeOk && !dlerror_msg.some((msg) => e.message.includes(msg)))
```

Comment:
replace `dlerror_msg` truthy check for a trivial fallback (Line 34) mostly for keeping line short.